internal/store/hpa.go

```diff
@@ -143,6 +143,53 @@ var (
 				}
 			}),
 		},
+		{
+			Name: "kube_hpa_status_current_average_value",
```

Comment:
@ashleykasim I didn't take a closer look at your change before. I think this would be a better name `kube_hpa_status_currentmetrics_average_value`.